source/application/REIXS/REIXSAppController.cpp

```diff
@@ -177,11 +160,6 @@ void REIXSAppController::setupScanConfigurations()
 	genericScanConfiguration_ = new AMGenericStepScanConfiguration();
 }
 
-void REIXSAppController::setupUserConfiguration()
```

Comment:
Did the setup for the REIXS user configuration happen elsewhere? Is it the same as the new CLS-level setup?